include/openssl/ssl.h.in

```diff
@@ -404,8 +404,9 @@ typedef int (*SSL_async_callback_fn)(SSL *s, void *arg);
 # define SSL_OP_NO_TLSv1_2                               SSL_OP_BIT(27)
 # define SSL_OP_NO_TLSv1_1                               SSL_OP_BIT(28)
 # define SSL_OP_NO_TLSv1_3                               SSL_OP_BIT(29)
-# define SSL_OP_NO_DTLSv1                                SSL_OP_BIT(26)
+# define SSL_OP_NO_DTLSv1                                SSL_OP_BIT(28)
```

Comment:
Please revert this, it would be an API change.

Reply:
Ok will do. But it looks like a defect since it looks like comparable tls and dtls versions should have the same op number.

Reply:
I do not think the bit numbers really matter much as soon as the values are stable.